packages/algob/src/lib/script-checkpoints.ts

```diff
@@ -18,6 +18,7 @@ import {
 } from "../types";
 
 export const scriptsDirectory = "scripts";
+export const testsDirectory = "test";
```

Comment:
Please add documentation regarding this (that test directory should be named "test" only, otherwise `algob test` will fail.

Reply:
Also rename `/sample-project/tests` -> `/sample-project/test`